src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/ReportController.kt

```diff
@@ -66,17 +66,15 @@ class ReportController(
 
     fun getRunnableReports(): List<ReportWithDate>
     {
-        val reports = orderlyServerAPI.get("/reports/source", context).listData(String::class.java)
+        val reports = orderlyServerAPI.getRunnableReportNames(context.queryParams())
```

Comment:
Just because same call to the same url is used from two places now (also in ReportController) so I thought preferable to have common method both could use, but you're right it's not in the same style as the others. Could perhaps have shared consts for the url that both use instead?